book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -0,0 +1,179 @@
+# Barriers to reproducibility
+
+So far we have [explained what we mean](overview-definitions) by reproducible research and explained some of the [additional benefits](overview-benefits).
+
+But there are many barriers to reproducible research.
+
+You can watch Kirstie Whitaker describe some of them in [her talk about _The Turing Way_](https://youtu.be/wZeoZaIV0VE?t=312) at [csv,conf,v4](https://csvconf.com/2019) in May 2019.
```

Comment:
Should we add it at the end of the chapter or in the image title where use a screenshot from here? At this point in the chapter, it's unsure what is a barrier and if a reader wants to indeed learn more about it.